src/main/java/hlf/java/rest/client/service/ControllerService.java

```diff
@@ -0,0 +1,7 @@
+package hlf.java.rest.client.service;
+
+import hlf.java.rest.client.model.FileUploadHandler;
+
+public interface ControllerService {
```

Comment:
The name 'ControllerService' seems to be very generic. Maybe some other name which actually mentions what this class does, like ConfigHandlerService

Reply:
changed to ConfigHandlerService